jitsucom-jitsu.yaml

```diff
@@ -83,12 +84,74 @@ subpackages:
           cp -r ./webapps/console/public/* ${{targets.contextdir}}/app/webapps/console/public
 
           # remove musl deps that end up polluting our SCA-generated dependencies with unresolveable deps.
-          rm -rf \
-              ${{targets.contextdir}}/app/node_modules/.pnpm/@img+sharp-libvips-linuxmusl-x64@1.0.4 \
-              ${{targets.contextdir}}/app/node_modules/.pnpm/@img+sharp-linuxmusl-x64@0.33.5
+          find ${{targets.contextdir}}/app/node_modules -type d -name '*musl*' -exec rm -rf {} + 2>/dev/null || true
+
+          # Remove any musl-related .node files
+          find ${{targets.contextdir}} -type f -name '*.node' -exec sh -c 'ldd "$1" 2>&1 | grep -q musl && rm -f "$1"' _ {} \; 2>/dev/null || true
```

Comment:
I believe you'll also want the following:

`find ${{targets.contextdir}} -type f -name '*.node' -exec sh -c 'ldd "$1" 2>&1 | grep -q libvips-cpp && rm -f "$1"' _ {} \; 2>/dev/null || true`

as we can see the following in the build log for jitsucom-jitsu-console:

```
INFO 🔗 generating package jitsucom-jitsu-console-2.10.0-r0
INFO 🔗 scanning for ld.so.conf.d files...
INFO 🔗 scanning for shared object dependencies...
INFO 🔗   found lib libvips-cpp.so.8.16.1 for app/node_modules/.pnpm/@img+sharp-linux-x64@0.34.1/node_modules/@img/sharp-linux-x64/lib/sharp-linux-x64.node
```